### DISCUSSION TITLE
Display DV data in probe info

### DISCUSSION THREAD

**Discussion by torbenott** [JingyiGF/HERBS#13] [Ideas]
Hi, I would suggest to add DV atlas coordinates of Insertion and Terminus coordinates when making a probe object. Currently only shows AP and ML data. I know I can back-calculate with the angle, etc., but would be nice to be displayed here I think.

**Comment by JingyiGF** [comment]
Hi, could you please give me a definition of DV according to your understanding? I was told that DV for insertion is always 0. So the DV shown in the popup window is the terminus DV. I am very sorry if it makes you confused. As I am not a neuroscientist, so I do exactly what I have been told by other neuroscientists. Please correct me if I am wrong. Many thanks for the suggestion.
